tests/spec/w3c/headers-spec.js

```diff
@@ -221,6 +221,25 @@ describe("W3C — Headers", function() {
     });
   });
 
+  describe("title", () => {
+    it("uses <h1> if already present", async () => {
+      const ops = makeStandardOps();
+      ops.body = "<h1 id='title'>Hello <code>World</code></h1>" + makeDefaultBody();
+      const doc = await makeRSDoc(ops);
+
+      const titleInHead = doc.querySelectorAll(".head .title.p-name");
+      expect(titleInHead.length).toEqual(1);
+
+      const title = doc.querySelectorAll(".title.p-name");
```

Comment:
Same here... might be good to clarify exactly what's being tested here. .

Reply:
``` js
const title = doc.querySelectorAll(".title.p-name");
expect(title.length).toEqual(1); // original h1#title is removed
```
not sure how can I make this clearer. any help?

Reply:
would this one do? https://gist.github.com/sidvishnoi/e60fa8b991c163339e38f6293458a310

Reply:
Yep, that one is good.